packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -137,7 +157,9 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
         }
 
         this.browserstackLocal = new BrowserstackLocalLauncher.Local()
+
         this._updateCaps(capabilities, 'local')
+        this._updateCaps(capabilities, 'localIdentifier', opts.localIdentifier)
```

Comment:
if localIdentifier is not passed in caps, we set to undefined.. pl just check once if it causes any unintended effect..

Reply:
tested w/o localIdentifier, not breaking as such and no localIdentifier added to cap.